### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by NoNegs**
Updated Footer, CinemaContainer,HomeComponent,Jumbotron and NavigationBar

### CODE REVIEW

File: nbs-cinema/src/component/NavigationBar.js [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
-    &:hoover{
-        color: #fff;
-     }
-   }
   `;
 
-
-
-export default class NavigationBar extends React.Component {
+export class NavigationBar extends Component {
```
Comment (ianowenQA) [inline nbs-cinema/src/component/NavigationBar.js]: Make this the default export

File: nbs-cinema/src/component/CinemaContainer.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ import Reviews from './ReviewBoard';
 import NoMatch from '../Pages/NotFound';
 import CommentBoard from './CommentBoard';
 import { Layout } from './Layout';
-import NavigationBar  from './NavigationBar';
+import { NavigationBar } from './NavigationBar';
```
Comment (ianowenQA) [inline nbs-cinema/src/component/CinemaContainer.js]: Export NavigationBar as default and revert this.

File: nbs-cinema/src/component/NavigationBar.js (hunk 1 above)
Comment (NoNegs) [reply nbs-cinema/src/component/NavigationBar.js]: done

File: nbs-cinema/src/component/CinemaContainer.js (hunk 2 above)
Comment (NoNegs) [reply nbs-cinema/src/component/CinemaContainer.js]: done

**Pull request merged**